src/cryptenroll/cryptenroll.c

```diff
@@ -82,6 +83,7 @@ static int help(void) {
                "\n%sEnroll a security token or authentication credential to a LUKS volume.%s\n\n"
                "  -h --help            Show this help\n"
                "     --version         Show package version\n"
+               "     --lock-with-pin   Require entering a PIN to unlock the volume (FIDO2)\n"
```

Comment:
should have "fido2" in its name, and i would prefer an explicit option for this:

--fido2-with-client-pin=yes|no